x/emissions/module/rewards/scores_test.go

```diff
@@ -0,0 +1,191 @@
+package rewards_test
+
+import (
+	"math"
+
+	cosmosMath "cosmossdk.io/math"
+	"github.com/allora-network/allora-chain/x/emissions/module/rewards"
+	"github.com/allora-network/allora-chain/x/emissions/types"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+func (s *RewardsTestSuite) TestGetReputersScores() {
+	topidId := uint64(1)
+	block := int64(1003)
+
+	// Generate reputers data for tests
+	reportedLosses, err := mockReputersScoresTestData(s, topidId, block)
+	s.Require().NoError(err)
+
+	// Generate new reputer scores
+	scores, err := rewards.GenerateReputerScores(
+		s.ctx,
+		s.emissionsKeeper,
+		topidId,
+		block,
+		reportedLosses,
+	)
+	s.Require().NoError(err)
+
+	expectedScores := []float64{17.98648, 20.32339, 26.44637, 11.17804, 14.93222}
```

Comment:
how did you calculate the expected scores? For these tests to be meaningful the scores would need to be calculated outside of the context of the tests, using the expected whitepaper behavior